ble/src/main/java/no/nordicsemi/android/ble/BleManager.java

```diff
@@ -2798,7 +2798,7 @@ private void nextRequest(final boolean force) {
 					break;
 				}
 				case DISCONNECT: {
-					result = internalDisconnect();
+					result = internalDisconnect(GattError.GATT_CONN_TERMINATE_PEER_USER);
```

Comment:
I think the status here should also be GattError.GATT_CONN_TERMINATE_LOCAL_HOST. 
Peer user is the other side (peripheral).

Reply:
Ah, ok